packages/backend/src/graphql/schema.gql-to-typescript.ts

```diff
@@ -1,5 +1,19 @@
+import paginate from '@/helpers/pagination'
+import App from '@/models/app'
+import Execution from '@/models/execution'
 import ExecutionStep from '@/models/execution-step'
+import Flow from '@/models/flow'
 import TableMetadata from '@/models/table-metadata'
 
-export type TableMetadataGraphQLType = TableMetadata
+export type AppGraphQLType = App
 export type ExecutionStepGraphQLType = ExecutionStep
+export type TableMetadataGraphQLType = TableMetadata
+
+// TODO: check if there is a better way to express connections
```

Comment:
can i check why is it necessary to express connections?

Reply:
We use Relay-ish connections in our queries (nodes & edges) so we need to type these. But I was hoping for these types to be auto-generated by codegen instead of me having to manually code it